lib/auth/apiserver.go

```diff
@@ -1137,14 +1138,33 @@ func (s *APIServer) getDomainName(auth ClientI, w http.ResponseWriter, r *http.R
 	return domain, nil
 }
 
+// LocalCAResponse contains a PEM-encoded  CA cert.
+type LocalCAResponse struct {
+	// TLSCA is a PEM-encoded TLS certificate authority.
+	TLSCA []byte `json:"tls_ca"`
+}
+
+// DEPRECATED: This will not work with HA HSM clusters. Prefer getClusterCACerts
 // getClusterCACert returns the CAs for the local cluster without signing keys.
 func (s *APIServer) getClusterCACert(auth ClientI, w http.ResponseWriter, r *http.Request, p httprouter.Params, version string) (interface{}, error) {
-	localCA, err := auth.GetClusterCACert()
+	certs, err := auth.GetClusterCACerts()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	return LocalCAResponse{
```

Comment:
Is this a backwards-compatible change? Seems like the format of the returned response is changing, no?